### ISSUE TITLE
Dark mode for everest UI 

### ISSUE THREAD

**Issue by Fiskepudding** [graphprotocol/mission-control-curator#191]
Would be really great with an actual dark mode toggle for the dAPP.

**Comment by yanivtal** [comment]
Yes this would be great. We used ThemeUI for this which supports themes. Just needs to be implemented
